src/app/loginComponents/accounts/account-sidebar/account-sidebar.component.ts

```diff
@@ -24,13 +26,16 @@ export class AccountSidebarComponent implements OnInit {
   public isRefreshing$: Observable<boolean>;
   public syncBadgeGit: boolean = false;
   public syncBadgeGoogle: boolean = false;
+  public showEmailWarning = false;
+  protected ngUnsubscribe: Subject<{}> = new Subject();
```

Comment:
1. This looks unused
2. If you do need it (still looking at code), you can have the component inherit from app/shared/base so you don't have to reimplement it.

Reply:
mm you're right I can remove this, thank you - will remove and push once any other changes are finalised